### DISCUSSION TITLE
Numbers of annotations per gene

### DISCUSSION THREAD

**Discussion by diatomsRcool** [geneontology/helpdesk#345] [Q&A]
Hello. I am interested in looking at the number of annotations per gene. Specifically, I would like a list of genes in GO with no or very few annotations. Any suggestions on how I might get started on this? Thanks!

**Comment by suzialeksander** [comment]
Hi @diatomsRcool, were you interested in a specific organism?

**Comment by diatomsRcool** [comment]
human

**Comment by suzialeksander** [comment]
Hi @diatomsRcool, 

Did you try using the files at https://ftp.ebi.ac.uk/pub/databases/GO/goa/UNIPROT/ ?  You should be able to filter by taxon, then sort by accession and compare to the reference proteome.  If you tried this and goa_uniprot_all.gaf wasn't what you were looking for, please let me know.

**Comment by llrs** [comment]
I know this is an old question, but it might help future users.

I have done this via R using the annotation provided in Bioconductor by the org.Hs.eg.db annotation package. 
The examples are in [the vignette](https://cran.r-project.org/web/packages/BaseSet/vignettes/advanced.html) of a package that can read *.obo files (if it helps for other users).
